cob_bringup/tools/base_collision_observer.launch

```diff
@@ -19,9 +19,9 @@
   <node pkg="cob_collision_velocity_filter" type="collision_velocity_filter" name="collision_velocity_filter" output="screen">
     <!-- remap incoming variables -->
     <remap from="~obstacles" to="/local_costmap_node/costmap/obstacles" />
-    <remap from="~teleop_twist" to="/base_controller/command_safe" />
+    <remap from="~teleop_twist" to="/base/twist_controller/command_safe" />
```

Comment:
I think it was a feature of the old teleop where base movement was stopped in case an obstacle entered the footprint...
@ipa-nhg is this feature still available with the new teleop?